### ISSUE TITLE
Non-working dokumentbeskrivelse relation for dokumentobjekt

### ISSUE THREAD

**Issue by petterreinholdtsen** [HiOA-ABI/nikita-noark5-core#85]
When trying to look up the list of mappe/saksmappe entries where a given documentobjekt is included, I ran into a problem.  The http://rel.kxml.no/noark5/v4/api/arkivstruktur/dokumentbeskrivelse/ relation point to http://localhost:8092/noark5v4/hateoas-api/arkivstruktur/dokumentobjekt/970a2d64-0193-4c61-aa89-d60eecf21c1b/dokumentbeskrivelse/, but that URL do not work.

In the development process I came across the http://rel.kxml.no/noark5/v4/api/arkivstruktur/registrering/ relation pointing to http://localhost:8092/noark5v4/hateoas-api/arkivstruktur/dokumentobjekt/970a2d64-0193-4c61-aa89-d60eecf21c1b/registrering/ which do not work either.  I hoped it could be used to jump past dokumenbeskrivelse directly to the registrering to get closer to relevant mappe entries, but no go.  Looking in the specification, I do not find this relation listed under part 7.2.1.7 (Dokumentobjekt) on page 100.  Is it a typo?

**Comment by petterreinholdtsen** [comment]
It occured to me, that perhaps the .../dokumentobjekt/<id>/dokumentbeskrivelse/ endpoint is useless?  The relation http://rel.kxml.no/noark5/v4/api/arkivstruktur/dokumentbeskrivelse/ could just as well point directly to the correct .../dokumentbeskrivelse/<id> endpoint.  This could be done all over the API, and reduce the amount of code required to implement the service (lots of endpoints are no longer needed).

Both approaches seem to be supported by the specification, as far as I can tell.  Perhaps it should be excplicity written in the spec to avoid the redundant endpoints?

**Comment by tsodring** [comment]
I agree. I haven't really thought that much about it. I tried to see if there was a need to have them as separate RELS, but I don't see that there is. You will save a lot of the GET requests by doing this. Maybe the 1..* relationships will have to follow the old style. A saksmappe can contain many journalpost.  The largest number of journalpost I have seen in a "normal" sakmappe is about 120. It was casefile for a job position and 120 people had applied. I've also seen "dummy" saksmappe that contain 1000s of journalposter, when a kommune decided to fix some of the data quality after a fagsystem integration where the software provider had just dumped journalpost in without creating a saksmappe.

**Comment by petterreinholdtsen** [comment]
I've started on a defect report asking for a clearer description of this in the spec.  See 2017-05-19-rel-direkte-href.md
